numpy/lib/npyio.py

```diff
@@ -798,21 +798,21 @@ def loadtxt(fname, dtype=float, comments='#', delimiter=None,
         strings will be decoded as 'latin1'. The default is whitespace.
     converters : dict, optional
         A dictionary mapping column number to a function that will convert
-        that column to a float.  E.g., if column 0 is a date string:
+        that column to a data-type.  E.g., if column 0 is a date string:
```

Comment:
instead of "that column", maybe "the string in that column"?

Reply:
@eric-wieser How about something similar to genfromtext function docs?  - "convert the data of a column to a value" + "of some data-type."

Reply:
Instead of "will convert..." perhaps "will parse the string into the desired column type"?

We need to make it clear that the function takes `unicode` or `bytes` as an argument (which one depends on the value of `encoding`)

Reply:
How about this? - "A dictionary mapping column number to a function that will parse the column string into the desired data-type value."

Reply:
Not even sure "data-type" is needed there